week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,32 +1,42 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  const greeting =
+    "Hello, my name is " + name + " and I am " + age + " years old";
+  return greeting;
+}
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+  remainder = a % b;
 
```

Comment:
Thanks, I found out that; 
It is not recommended to declare a variable without var/const keyword.
Scope of the variables declared without var keyword become global irrespective of where it is declared. Global variables can be accessed from anywhere in the web page. It can accidentally overwrite an existing global variable.